cmd/tether/attach.go

```diff
@@ -419,9 +436,12 @@ func (t *attachServerSSH) run() error {
 
 				channel.Close()
 
-				serverConn.Lock()
-				serverConn.ServerConn = nil
-				serverConn.Unlock()
+				t.serverConn.Lock()
+				// set serverConn to nil if not an exec session
+				if _, ok = t.config.Sessions[sessionid]; ok {
```

Comment:
so that we can see whether the connection is established or not, this value being nil is the looping condition